### ISSUE TITLE
Plugins Sprint 190 (2020-09-16 to 2020-10-06)

### ISSUE THREAD

**Issue by ericwill** [eclipse-che/che#17832]
## Sprint goals
1. Bugs (upstream and CRW)
2. Maintenance / Automation / Technical Debt

### Epics

**Separate vscode-java version from development target #16851** @tsmaeder 
* [x] Provide experimental "portable JDK" devfile #17924
    * Blocked by #17102
* [x] Support object array preferences in a devfile #17481

### CRW
* [x] Che throws Failed to execute Che command: Protocol "http:" not supported. Expected "https:" [CRW-1178](https://issues.redhat.com/browse/CRW-1178) @svor
    * Investigated, looks like a CRW config issue
* [x] Failed to launch debug session in the "Go" devfile [CRW-1021](https://issues.redhat.com/browse/CRW-1021) @svor 
* [x] Support kerberos authentication for private git repos #17782 @vitaliy-guliy 
* [x] It's not obvious to users that after installing php dependencies, browser refresh is required [CRW-981](https://issues.redhat.com/browse/CRW-981) @svor  
* [x] Openshift Connector plugin - inClusterUser does not have appropriate rights [CRW-1158](https://issues.redhat.com/browse/CRW-1158) @vinokurig 

### Bugs
* [ ] Align the GitHub authentication plugin with the latest vscode GitHub PR plugin #17684 @vinokurig 
    * Depends on missing API
* [x] PHP autocomplete doesn't work with latest Intelephense #17493 @ericwill 
* [ ] Workspace Resource Monitor Plugin #17205 @svor 

### Unplanned
* [x] Failed to start Java LS (JAVA_HOME and PATH refer to JRE, not JDK) [CRW-1224](https://issues.redhat.com/browse/CRW-1224)
* [x] Failed to start debug session for the "JavaSpringBoot" devfile [CRW-1234](https://issues.redhat.com/browse/CRW-1234)
* [x] Failed to start debug session for the "Python" devfile [CRW-1235](https://issues.redhat.com/browse/CRW-1235)
    * Investigation shows the issue is caused by migration to Node 12, and will be resolved with the latest Python extension
* [x] Update vale binary version in the che-sidecar-vale image #18008 @ericwill 

### Open PRs From Last Sprint
* [ ] Don't open Welcome page if already open (was: Too many Welcome pages) [CRW-1029](https://issues.redhat.com/browse/CRW-1029) @vitaliy-guliy 
    * [x] new welcome page every time when I refresh my workspace #16352

#### Theia API
* [ ] Add environmentVariableCollection to PluginContext eclipse-theia/theia#8405 @vinokurig 
* [x] Apply CommentController plugin API eclipse-theia/theia#8492 @vinokurig
* [x] Support 'onStartupFinished' plugin activation event eclipse-theia/theia#8488 @vinokurig

#### Automation / Tests
* [x] Add plugin-registry PR check to validate new/updated vsx #17027 @ericwill 
    * [x] Check upstream GH repo against revision to make sure revisions are correct/exist
* [ ] Automate updating of sidecar images #17641 @ericwill 
    * [ ] Experiment with a GH action che-dockerfiles that opens a PR on the che-plugin-registry

#### Maintenance
* [ ] Plugin Updates
    * [ ] python needs to be updated to 2020.9.111407 #16183 @tsmaeder 
    * [ ] sonarlint-vscode 1.16 -> 1.17 @ericwill 
    * [x] vscode-intelephense-client 1.3.11 -> 1.5.4 @ericwill 
    * [x] OpenShift connector 0.1.5 -> 0.1.7 @ericwill 
        * Postponed to next sprint for the OpenShift connector 0.2.0 release
* [x] Upgrade forked vscode-git to the latest version #16041 @vinokurig 

#### Availability
| Name  | Availability | 0.6x |
| ------------- | ------------- | ------------- |
| @vitaliy-guliy  | 5  | 3 |
| @vinokurig  | 15  | 9 |
| @ericwill | 14 | 9 |
| @tsmaeder | 12 | 7 |
| @svor | 15 | 9 |

**Comment by gsmachado** [comment]
Hello guys,
I know this might be on the radar for this sprint, but check out this comment:
https://github.com/eclipse-theia/theia/issues/7185#issuecomment-703184365
Let me know!

**Comment by ericwill** [comment]
Superseded by #18016
